packages/sdk/src/types/Campaigns.ts

```diff
@@ -124,37 +434,59 @@ export type CampaignsAddCertainVoucherParams = Pick<
 	'category' | 'redemption' | 'metadata' | 'additional_info'
 >
 
-export type CampaignsAddVoucherResponse = Pick<
-	VouchersResponse,
-	| 'code'
-	| 'object'
-	| 'campaign'
-	| 'category'
-	| 'created_at'
-	| 'type'
-	| 'discount'
-	| 'gift'
-	| 'start_date'
-	| 'expiration_date'
-	| 'publish'
-	| 'redemption'
-	| 'active'
-	| 'additional_info'
-	| 'metadata'
-	| 'assets'
->
+export type CampaignsAddVoucherResponse = VoucherObject | AsyncActionCreateResponse
 
 export type CampaignsAddCertainVoucherResponse = CampaignsAddVoucherResponse
 
-export type CampaignsImportVouchers = Pick<
-	VouchersImport,
-	'code' | 'redemption' | 'metadata' | 'additional_info' | 'active'
->
+export interface AddVouchersToCampaign {
```

Comment:
All of these fields are not requried to add voucher to campaign.
Only code is required